src/approaches/bilevel_planning_approach.py

```diff
@@ -105,3 +105,7 @@ def _get_current_predicates(self) -> Set[Predicate]:
         Defaults to initial predicates.
         """
         return self._initial_predicates
+
+    def get_option_model(self) -> _OptionModelBase:
```

Comment:
move this to `oracle_approach.py` and clarify that this method is only allowed for the oracle approach (you can copy the docstring of `get_last_plan` and lightly edit it)